src/main/java/com/example/parser/repository/PermitRepository.java

```diff
@@ -2,15 +2,12 @@
 
 import com.example.parser.entity.PermitForEmissionsOfPollutants;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.repository.CrudRepository;
 import org.springframework.stereotype.Repository;
 
 import java.util.List;
 
 @Repository
 public interface PermitRepository extends JpaRepository<PermitForEmissionsOfPollutants, Long> {
```

Comment:
Looks like this class has no code and can be deleted

Reply:
done